draft-ietf-core-groupcomm-bis.md

```diff
@@ -1003,33 +1009,41 @@ Thus, consistent with {{Section 7 of RFC7641}}, a server in a CoAP group MUST st
+
+It follows that, in either case, this replay attack would not accomplish anything on the client, although it does effectively target the servers in the group.
 
 If Group OSCORE is used, such a replay attack on the servers is prevented, since a client protects each different request with a different Sequence Number value, which is in turn included as Partial IV in the protected message and takes part in the construction of the AEAD cipher nonce. Thus, a server would be able to detect the replayed request, by checking the conveyed Partial IV against its own replay window in the OSCORE Recipient Context associated with the client.
 
-This requires a server to have a synchronized, up-to-date view of the sequence number used by the client. If such synchronization is lost, e.g., due to a reboot, or suspected so, the server should use the challenge-response synchronization method based on the Echo Option for CoAP defined in {{RFC9175}} as described in {{Section 10 of I-D.ietf-core-oscore-groupcomm}}, in order to (re-)synchronize with the client's sequence number.
+This requires a server to have a Replay Window that is in a valid state. If the server's Replay Window is initialized as invalid, e.g., due to a reboot, the server should use the challenge-response synchronization method based on the Echo Option for CoAP defined in {{RFC9175}} as described in {{Section 9 of I-D.ietf-core-oscore-groupcomm}}, in order to make the Replay Window valid before resuming to accept incoming messages from other group members.
 
 ## Use of CoAP No-Response Option ##
 
 When CoAP group communication is used in CoAP NoSec (No Security)
 mode (see {{chap-unsecured-groupcomm}}), the CoAP No-Response Option {{RFC7967}} could be misused by a malicious client to evoke as many responses from servers to a group request as possible, by using the value '0' -- Interested in all responses. This might even override the default behavior of a CoAP server to suppress the response in case there is nothing of interest to respond with. Therefore, this option can be used to perform an amplification attack (see {{ssec-amplification}}).
 
-A proposed mitigation is to only allow this option to relax the standard suppression rules for a resource in case the option is sent by an authenticated client. If sent by an unauthenticated client, the option can be used to expand the classes of responses suppressed compared to the default rules but not to reduce the classes of responses suppressed.
+As a mitigation, a server that takes into account the No-Response Option relaxes the standard suppression rules for a resource only in case the option is sent by an authenticated client. Consequently, if sent by an unauthenticated client, the option can still be used to expand the classes of responses suppressed compared to the default rules, but not to reduce the classes of responses suppressed.
```

Comment:
Not sure that indicative mood ("relaxes") works here.

» In grammar, the indicative mood is a verb mood that the speaker or writer uses to express information that sounds factual. «

Reply:
Changed to "can specifically relax"